packages/styles/stories/introduction.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Styles/About the Style injection system" />
```

Comment:
Probably need some background information around what is "style injection", what's common solution out there, pro/cons, etc.